### ISSUE TITLE
Makespan of ta41 cannot be achieved as mentioned in the paper

### ISSUE THREAD

**Issue by Iamfxz** [prosysscience/RL-Job-Shop-Scheduling#33]
when I run the code in master branch, I get the follow result:
![image](https://user-images.githubusercontent.com/39674855/200323766-61e18a23-d919-4303-8a69-b30362fdd8d8.png)
how to get the result of paper: 2208
![image](https://user-images.githubusercontent.com/39674855/200323983-cff5525c-0984-4e80-aaec-e5788b98b813.png)

**Comment by Iamfxz** [comment]
more training time:
![image](https://user-images.githubusercontent.com/39674855/200330889-a14001af-37b3-4c22-8c66-fd740da4857e.png)

**Comment by ingambe** [comment]
Hi, thanks for trying to reproduce :)

As we are training from scratch the neural network and setting a timeout of 10 minutes, the results of the approach will depend on the hardware it is run on.
It looks like you are using a CPU-only platform. Using a GPU is much faster. Also, the more core the machine has, the better/faster.

To give you an example of a recent run, I was able to reproduce using a different machine with an NVIDIA RTX A5000 and an AMD epyc CPU with 96 cores.
As you can see from the W&B logging platform, we manage good results for ta60 in 10 minutes: https://wandb.ai/ingambe/RL-Job-Shop-Scheduling-JSS/runs/1gq3knc0/overview

If your machine is less powerful, you might have to run longers/with more environments per worker.
What is your configuration?

**Comment by WayneHsiao0225** [comment]
Does the "step" on the y-axis means the "episode"?

**Comment by ingambe** [comment]
No, it refers to the number of training iterations
